e2e/playwright/tests/legal_hold_plugin.spec.ts

```diff
@@ -0,0 +1,39 @@
+import {test, expect} from '@e2e-support/test_fixture';
+import LegalHoldPluginPage from '../pages/legal_hold_plugin';
+
+let legalHoldPluginPage: LegalHoldPluginPage;
+
+test('create a legal hold successfully', async ({pw, pages}) => {
+    //login as admin user
```

Comment:
nit: please make commenting format consistent, ex. a space after the `//`
```suggestion
    // login as admin user
```